Code/Framework/AzFramework/Tests/Platform/Common/Xcb/azframework_xcb_tests_files.cmake

```diff
@@ -0,0 +1,16 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(FILES
+    ../../../../../AzCore/Tests/Main.cpp
```

Comment:
mmm, instead, add your own main. You also dont have benchmarks in this so there is no reason to be using that...
You can just do a main that has:
```
AZ_UNIT_TEST_HOOK(DEFAULT_UNIT_TEST_ENV);
```

Moreover, you can put that in any other cpp file. I would still recommend to create a `main.cpp` file since is easier to find where the entry point is.

Reply:
Sounds good. I copied this from `Code/Framework/AzFramework/Tests/frameworktests_files.cmake`, should I update that file as well?

Reply:
Should I make both:
```
Code/Framework/AzFramework/Tests/Main.cpp
Code/Framework/AzFramework/Tests/Platform/Common/Xcb/Main.cpp
```
with the same contents? Or just the first file and include it in both targets?

Reply:
yes, that would be great, thanks!

Reply:
I went ahead and made separate files.